checkov/terraform/checks/provider/aws/defaulttags.py

```diff
@@ -0,0 +1,28 @@
+import re
```

Comment:
this import is unused and should be removed

Reply:
Removed.  Thanks!